src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.GenerateAssemblyInfo.targets

```diff
@@ -123,6 +123,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     
     <PropertyGroup>
       <FileVersion Condition="'$(FileVersion)' == ''">$(AssemblyVersion)</FileVersion>
+      <InformationalVersion Condition="'$(InformationalVersion)' == ''">$(AssemblyVersion)</InformationalVersion>
```

Comment:
The value used to come straight from $(Version), but this is changing the default to use $(AssemblyVersion). Is that intentional?